website/src/data/components/choropleth/props.js

```diff
@@ -65,6 +65,15 @@ const props = [
         controlType: 'quantizeColors',
         group: 'Style',
     },
+    {
+        key: 'domain',
+        help: 'Defines uppper and lower bounds of color shading',
+        type: 'number[]',
+        required: true,
+        defaultValue: ChoroplethDefaultProps.domain,
+        controlType: 'domainColors',
```

Comment:
There is not such control unfortunately, you can remove this key, I'll add it later if I add one

Reply:
Removed.